aspnetcore/host-and-deploy/windows-service/samples/2.x/AspNetCoreService/README.md

```diff
@@ -1,3 +1,5 @@
-# ASP.NET Core Windows Service Sample (Framework-dependent Deployment)
+# ASP.NET Core Windows Service Sample
 
 This sample shows how to host an ASP.NET Core app as a Windows Service without using IIS. This sample demonstrates the scenario described in [Host ASP.NET Core in a Windows Service](https://docs.microsoft.com/aspnet/core/host-and-deploy/windows-service).
+
+After establishing and starting the service, access the app in a browser at `http://localhost:5000/`.
```

Comment:
#14193 asked to show how to configure a port, not about the default. The default exists for dev purposes, published apps should always configure a url/port.

Reply:
I *almost* read it that way, but he shows "localhost" in the ask and ...

> so the service which is running can be navigated to through a browser.

... implies that he had exactly that problem ... couldn't get it running on the default and seemed to be wondering if it would run at all without an explicit config. He didn't elaborate, and I may have guessed wrong.

WRT an enhancement to the topic for this and since endpoint config is a Kestrel thing, shouldn't we just cross-link to the Kestrel topic endpoint config section and explicitly recommend setting a URL/port as opposed to showing an opinionated example here? I'd like to avoid content duplication and just get the reader to the right spot to see all of the knobs.

Reply:
Showing even the simplified ASPNETCORE_URLS example would be a good start. And yes linking to Kestrel and HttpSys for advanced config would help too.

Reply:
@Tratcher guessed my intentions correctly. Changing localhost to an IP made it available publicly too, perhaps a small sample + reference for kestrel configuration is the safest way here. then again this was already merged so  i guess it is done anyway.

Reply:
@taori ... access the Kestrel endpoint config content. There are **_many knobs_** for those options.

Reply:
@guardrex Yes i know. But it's a guide to make a standalone service. It only makes sense to reference the contents of most common concerns a developer might have when reading that article, which in this case among other things is "where do i configure my ip so my service can be talked to". As you can see i did find the info myself after some research - just trying to ease the use the documentation for the next developer really.